src/plugins/Multipart.js

```diff
@@ -12,9 +12,14 @@ module.exports = class Multipart extends Plugin {
     // Default options
     const defaultOptions = {
       fieldName: 'files[]',
+      fields: null,
       responseUrlFieldName: 'url',
       bundle: true,
-      headers: {}
+      headers: {},
+      getUploadUrl (xhr) {
+        const resp = JSON.parse(xhr.response)
+        return resp[this.responseUrlFieldName]
+      }
```

Comment:
This option allows the S3 plugin to take the uploaded url from S3's XML response